pkg/container/batch/batch.go

```diff
@@ -147,6 +147,32 @@ func (bat *Batch) ExpandNulls() {
 	}
 }
 
+func (bat *Batch) AntiShrink(sels []int64) {
```

Comment:
What is an AntiShrink?

Reply:
A Shrink is used to perserve the rows with sels row number, and the AntiShrink us used to perserve the rows whose row number is not in sels.